docker/Dockerfile

```
@@ -25,23 +27,30 @@ RUN chown -R espressopp:espressopp /home/espressopp/espressopp /home/espressopp/
 USER espressopp
 
 WORKDIR /home/espressopp/espressopp
-RUN mkdir build
+RUN maintainer/check_codestyle.sh ${CODESTYLE} ${TRAVIS_BRANCH} ${TRAVIS_PULL_REQUEST} ${TRAVIS_COMMIT} ${TRAVIS_COMMIT_RANGE}
 
-WORKDIR build
-RUN ccache -z
-RUN cmake -DCMAKE_INSTALL_PREFIX=/usr -DEXTERNAL_BOOST=$EXTERNAL -DEXTERNAL_MPI4PY=$EXTERNAL -DWITH_XTC=ON ${COVERAGE:+-DUSE_GCOV=ON} ..
-RUN make VERBOSE=1 -j2 all ${BUILD_UG:+ug doc ug-pdf}
-RUN ccache -s
-RUN make test CTEST_OUTPUT_ON_FAILURE=1
-RUN make install DESTDIR=${PWD}
-RUN cd .. && if [ ${COVERAGE} ]; then \
-  if [ ${CC} = clang ]; then \
-    $HOME/.local/bin/codecov --gcov-exec "llvm-cov gcov"; \
+RUN if [ "${CODESTYLE}" = ON ]; then \
```

```suggestion
RUN if [ "${CODESTYLE}" != ON ]; then \
```
and remove the `else` part!